src/components/swap-history/index.tsx

```diff
@@ -112,14 +112,45 @@ const Transaction: FC<{ address: string; transaction: TransactionProps }> = ({
 const Component: FC = () => {
   const { t } = useTranslation();
   const { address = "", isConnected } = useAccount();
-  const { transactions } = useSwapHistory();
+  const { transactions, clearHistory } = useSwapHistory();
 
   return isConnected ? (
     <div className="swap-history">
-      <span className="heading">{t(constantKeys.TRANSACTIONS)}</span>
-      {transactions.map((transaction, index) => (
-        <Transaction key={index} address={address} transaction={transaction} />
-      ))}
+      <div className="history-header">
+        <span className="heading">{t(constantKeys.TRANSACTIONS)}</span>
+        {transactions.length > 0 && (
+          <button 
```

Comment:
Most of the time, we use a `<span>` tag for buttons so that we don't need to add custom CSS. For example:

<img width="455" alt="image" src="https://github.com/user-attachments/assets/6165f25e-05c0-4f8d-b2c5-7a73a202af49" />